gci/views.py

```diff
@@ -28,4 +30,26 @@ def index(request):
                  '<a href="https://github.com/{username}">{username}</a>'
                  .format(student_url=student_url, student_id=student_id,
                          username=username))
+    timestamp = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+    s.append('</ul><i id="time" data-time="{unix}">'
+             'Last updated: {timestamp} (<span id="ago"></span>)</i>'
+             .format(unix=timegm(datetime.utcnow().utctimetuple()),
+                     timestamp=timestamp))
+    s.append('<script>'
```

Comment:
Move the script into a static 'asset' `.js` file , which is published to the deployed site, and use `<script src="...">`

Reply:
Where should the asset file go? Do I also need to write something in the deploy.sh to copy over these asset files?

Reply:
Nevermind, Django made this much easier than I expected 👍